policy/modules/contrib/ldap.fc

```diff
@@ -13,11 +13,8 @@
 /usr/lib/slapd	--	gen_context(system_u:object_r:slapd_exec_t,s0)
 
 /var/lib/ldap(/.*)?	gen_context(system_u:object_r:slapd_db_t,s0)
-/var/lib/ldap/replog(/.*)?	gen_context(system_u:object_r:slapd_replog_t,s0)
```

Comment:
You should then probably also remove all references to "slapd_replog_t" tree-wide.

Reply:
grep -r slapd_replog_t